treeherder/model/migrations/0038_commit_search_vector_idx.py

```diff
@@ -1,8 +1,7 @@
-# Generated by Django 5.1.5 on 2025-02-27 18:06
```

Comment:
The migration has been applied, a new migration is needed.

Reply:
I rolled back to the old migration `0037_bugjob...` then deleted my `0038_commit_search..` migration. so when I ran `makemigration` and another one got generated, I renamed it to `0038_commit_search..` and applied migration.  Would that still be a problem?

Reply:
A migration 0039 landed which depends on the old 0038. Please use a new migration 0040.

Reply:
Okay got it

Reply:
I have created a new migration for this